### ISSUE TITLE
error "GoogleApiBrowser doesNotUnderstand #sortAPIList

### ISSUE THREAD

**Issue by jmari** [seandenigris/St-Google-API#6]
trying to open GoogleApiBrowser. #readApiDef: sends this message at the end of the method and it doesn't exist.

**Comment by psvensson** [comment]
Yes, it looks like there has been some WIP commit at some point, where the methdo for some reason was not implemented (or not committed). I commented it out in a recent PR but will actually implement it and use it soon :)
